### DISCUSSION TITLE
Laravel schedule not run

### DISCUSSION THREAD

**Discussion by utqinadhif** [laravel/framework#50503] [Q&A] [answered]
### Laravel Version

11.0.3

### PHP Version

8.3

### Database Driver & Version

Postgres

### Description

In laravel 10 schedule config is read from kernel.php, in laravel 11, i read from the doc, config must write in routes/console.php.

In php artisan schedule:list, all my schedule listed, but when php artisan schedule:run, schedule not work, 

Whats wrong about me?

### Steps To Reproduce

Please help me

**Comment by crynobone** [comment, marked as answer]
Hi there,

Thanks for reporting the problem you are encountering, but it looks like this is a question which may be better suited for a support channel. We only use this issue tracker for reporting bugs with the library itself. If you have a question on how to use functionality provided by this repository you can try one of the following channels:

* [Laracasts Forums](https://laracasts.com/discuss)
* [Laravel.io Forums](https://laravel.io/forum)
* [StackOverflow](https://stackoverflow.com/questions/tagged/laravel)
* [Discord](https://discordapp.com/invite/KxwQuKb)
* [Larachat](https://larachat.co/)
* [IRC](https://web.libera.chat/?nick=laravelnewbie&channels=#laravel)

However, this issue will not be locked and everyone is still free to discuss solutions to your problem!

Thanks.

**Comment by Kovah** [comment]
Hey @utqinadhif, I just hade the same problem and noticed, that my bootstrap/app.php file was missing the needed configuration. It should look like this:
```php
return Application::configure(basePath: dirname(__DIR__))
    ->withProviders()
    ->withRouting(
        web: __DIR__ . '/../routes/web.php',
        api: __DIR__ . '/../routes/api.php',
        commands: __DIR__ . '/../routes/console.php', // <<< important line
        channels: __DIR__ . '/../routes/channels.php',
        health: '/_healthcheck',
    )...
```
